ui/js/controllers/jobs.js

```diff
@@ -109,6 +109,54 @@ treeherderApp.controller('ResultSetCtrl', [
             $rootScope.selectedJob = job;
         };
 
+        $rootScope.$on(thEvents.applyNewJobs, function (ev, resultSetId) {
+            if ($scope.resultset.id === resultSetId) {
+                $scope.getDecisionTaskVisibility();
+            }
+        });
+
+        $scope.getDecisionTaskVisibility = function () {
+            let dtId = ThResultSetStore.getGeckoDecisionTaskId(
+                $scope.repoName,
+                $scope.resultset.id
+            ).then(function () {
+                if (dtId.$$state.value === 'No decision task') {
+                    $scope.isDecisionTaskVisible = false;
+                } else {
+                    $scope.isDecisionTaskVisible = true;
+                }
+            }, function (e) {
+                console.log(e);
```

Comment:
So this `.then()` appears to quiet the Angular 1.6 `Possibly unhandled rejection` errors to the console. If I've done it correctly, my question is should I write the error message to ThLog? I'm only writing it to the console in my WIP commit at the moment to see what is happening.